src/bigtiffimage.cpp

```diff
@@ -1,14 +1,14 @@
 #include "bigtiffimage.hpp"
 
-#include "safe_op.hpp"
-#include "exif.hpp"
-#include "error.hpp"
-#include "image_int.hpp"
-#include "enforce.hpp"
-
 #include <cassert>
-#include <limits>
 #include <iostream>
+#include <limits>
+
+#include "enforce.hpp"
```

Comment:
I also see here a problem. I think it is not written anywhere, but it was discussed in the past between the main contributors. We agreed on using the following style:

```
// Include exiv2 headers 
#include "enforce.hpp"
...

// Include 3rd party headers
...

// Include C++ and STL headers
...
```

I know that the situation is not perfect at the moment, but we do not want to break that sorting where we have it. 

In order to achieve that we would need to do some manual editing and introduce line breaks here and there.